app/labor/unread_notifications_emailer.rb

```diff
@@ -9,7 +9,6 @@ def self.send_all_emails(num = 1000)
     users.find_each do |user|
       UnreadNotificationsEmailer.new(user).send_email_if_appropriate
     rescue StandardError => e
-      logger = Logger.new(STDOUT)
```

Comment:
If you look closely this `logger` is passed as an argument in the line after it, to the `Airbrake::AirbrakeLogger.new` method. You can see why it's done like that here: https://github.com/airbrake/airbrake/blob/master/README.md#logger